include/seqan3/alphabet/composition/union_composition.hpp

```diff
@@ -59,8 +59,7 @@ namespace seqan3
  * \implements seqan3::alphabet_concept
  *
  * The union alphabet represents the union of two or more alphabets (e.g. the
- * four letter DNA alphabet + the gap alphabet). Note that you cannot assign
- * regular characters, but additional functions for this are available.
+ * four letter DNA alphabet + the gap alphabet).
```

Comment:
I spoke with @eseiler said that he removed this here, because this class has no own `::A, ::C` members.

This is fine with me.

Reply:
but "the additional functions" refers to `assign_char` and this alphabet has `assign_char` :thinking: